logs/logs.go

```diff
@@ -92,7 +93,12 @@ func (l *LogAgent) Run(ctx context.Context) {
 
 	t := time.NewTicker(time.Second)
 	defer t.Stop()
+	last := time.Now()
 	for {
+		if time.Since(last) >= 1 * time.Minute {
+			last = time.Now()
+			log.Printf("I! [logagent] open file count, %v", tail.OpenFileCount())
+		}
```

Comment:
Part of me feels like this would be cleaner by introducing another timer at a minute frequency. Thoughts?

Reply:
Agreed.
Will fix.